docs/release-notes/rockcraft-1-7-0.rst

```diff
@@ -0,0 +1,147 @@
+.. _release-1.7.0:
+
+Rockcraft 1.7.0 release notes
+=============================
+
+20 December 2024
+
+Learn about the new features, changes, and fixes introduced in Rockcraft 1.7.0.
+For information about the Rockcraft release cycle, see the
+:ref:`release_policy_and_schedule`.
+
+
+Requirements and compatibility
+------------------------------
+
+To run Rockcraft, a system requires the following minimum hardware and
+installed software. These requirements apply to local hosts as well as VMs and
+container hosts.
+
+
+Minimum hardware requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- AMD64, ARM64, ARMv7-M, RISC-V 64-bit, PowerPC 64-bit little-endian, or S390x
+  processor
+- 2GB RAM
+- 10GB available storage space
+- Internet access for remote software sources and the Snap Store
+
+
+Platform requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+  :header-rows: 1
+  :widths: 1 3 3
+
+  * - Platform
+    - Version
+    - Software requirements
+  * - GNU/Linux
+    - Popular distributions that ship with systemd and are `compatible with
+      snapd <https://snapcraft.io/docs/installing-snapd>`_
+    - systemd
+
+
+What's new
+----------
+
+Rockcraft 1.7.0 brings the following features, integrations, and improvements.
+
+
+Support for remote builds
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We added support for remote builds, which leverage off-site compute for
+building rocks and can build for multiple architectures with a single command.
+Remote builds are hosted on Launchpad.
+
+This feature is experimental and subject to change, and more information is
+available by running ``rockcraft remote-build --help``. Feel free to provide
```

Comment:
Adding a note to myself here that once #778 is merged I can update these to point to the new docs